lib/B/Deparse.t

```diff
@@ -3396,3 +3396,7 @@ $x //= exit 42;
 my $x = __LINE__;
 my $y = __FILE__;
 my $z = __PACKAGE__;
+####
+CORE::state sub FOO () { 42 }
+use feature 'state';
```

Comment:
Why include this line if you're only using `CORE::state`?

Reply:
I was just matching the text in the test failure. I've fixed it so the `CORE::` isn't needed.